CovertInfiltration/Localization/XComGame.int

```diff
@@ -351,18 +351,14 @@ ConsoleObjectiveTextPools[0]=
 ConsoleObjectiveTextPools[1]=
 
 [GatecrasherCI X2MissionTemplate]
-DisplayName="Operation Gatecrasher"
-Briefing="ADVENT has brought a pair of dissident academics to this facility for 're-education'. These two could be of extreme value to the Resistance. Ensure they both make it out of here alive."
+DisplayName="Rescue VIPs from ADVENT Compound"
```

Comment:
I think you should keep the `DisplayName="Operation Gatecrasher"` part

Reply:
All the other mission locs make both DisplayName and Briefing the exact same thing

Reply:
Uhhh... not all

```ini
[CovertEscape X2MissionTemplate]
DisplayName="Escape Covert Action"
Briefing="Covert Action - Escape ADVENT Counterops Team"

[SupplyExtraction X2MissionTemplate]
DisplayName="Extract ADVENT Supplies"
Briefing="Supply Raid - Procure ADVENT Supplies"

[NeutralizeFieldCommander X2MissionTemplate]
DisplayName="Neutralize Field Commander"
Briefing="Neutralize ADVENT Field Commander"
```